unpub/bin/unpub.dart

```diff
@@ -11,13 +11,15 @@ main(List<String> args) async {
   parser.addOption('database',
       abbr: 'd', defaultsTo: 'mongodb://localhost:27017/dart_pub');
   parser.addOption('proxy-origin', abbr: 'o', defaultsTo: '');
+  parser.addOption('opaque-token');
```

Comment:
This will be accepted as an optional CLI argument when running the server:
`dart run bin/unpub.dart --opaque-token=someStringValue`
or when running in production
`/app/bin/unpub --opaque-token=someStringValue`